Style object now gets passed to the rendered dom element

Code47X opened the pull request:
Cleaned up my PR a bit based on your suggestions.

Users can now use the style prop on ScrollAnimation components. The style object will be passed to the rendered dom element.

So you can now style ScrollAnimations inline if you wish by doing:
`<ScrollAnimation animateIn="fadeIn" duration={0.5}  style={{ color: 'red' }}>`

dbramwell commented:
Nice work. The tests are all still passing so I assume it still works if this.props.style is undefined?

Might be good to add a simple test case rendering a scrollanimation with the style prop set and then expecting scrollanimation.node.style.display to be inline or whatever.

Also if you could add a bit to the readme that'd be good.

Code47X commented:
I tested it on my portfolio site, which had ScrollAnimations both with and without the style attribute. I didn't see any problems.

Code47X commented:
Updated the README. I added the description for the style prop, and updated the changelog for version 2.0.1 and 2.0.2 (Assuming you want to up the version again for this change).

The testing framework was a little over my head, please feel free to add in a simple test for this prop if you wish.

Code47X commented:
I spoke too soon, added in a karma test for the style prop. It was successful when I ran the test, but please look over it and make sure the logic is testing the right thing.

dbramwell commented:
Nice job! Will merge and do a release :)

Thanks for your effort!
